### ISSUE TITLE
Equiping 'webbing' when wearing 'electronic jumpsuit' causes the item to glitch

### ISSUE THREAD

**Issue by CodePanter** [Aurorastation/Aurora.3#12204]
![webbing](https://user-images.githubusercontent.com/3697004/126021290-2af89824-b322-4383-8dca-761f0e3a0931.PNG)

I play an engineer, and I tried out the 'electronic jumpsuit' loadout item. When I have this equipped, trying to equip a 'webbing' or 'brown webbing vest' makes the item glitch out. 

As you can see in the screenshot, my right hand holds a 'webbing' item, and my left hand a 'brown webbing vest'. After pressing 'E' to equip them, my hands are, for all intents and purposes, empty. The webbing wasn't equipped, and you can still see it in my hand, but I am now once again able to pick up new items and interact with them. You see this in the screenshot, my right hand contains a pill, that was just taken from the bottle in the left hand.

In short, I expect to be able to equip webbing items as normal, and not have them disappear while still showing up in my hand slots.
